docs/developer-guides/merkle-root-construction.md

```diff
@@ -2,10 +2,52 @@
 
 ---
 
-#### Merkle Trees
+This page explains what merkle trees are, how Decred uses them and provides step-by-step instructions for constructing compliant merkle trees.
```

Comment:
On a slightly related note, while it isn't done consistently, the M really should be capitalized because a Merkle tree is named after its inventor, Ralph Merkle, and hence is a proper name.

Reply:
Have capitalized Merkle here and also on a couple other pages that aren't about to be [removed](https://github.com/jholdstock/dcrdevdocs/pull/43/commits/c464fbfd765de74900de42dea3fc22128b5178ac) anyway. @jholdstock hope that isn't too messy, can break out into separate PR if need be.

To be consistent, also submitted a [PR](https://github.com/decred/dcrdocs/pull/1013) to dcrdocs capitalizing Merkle there (where we're currently using a mix of both). 

@davecgh, is there anything else you see on this page?